dev-haskell/zip/zip-2.1.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.9.0.0.9999
+
+CABAL_HACKAGE_REVISION=2
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Operations on zip archives"
+HOMEPAGE="https://github.com/mrkkrp/zip"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="dev disable-bzip2 disable-zstd"
```

Comment:
These should be changed to `bzip2` and `zstd`, respectively. (The logic in `src_configure` will also have to be reversed)

Reply:
Something like:

```ebuild
IUSE="dev +bzip2 +zstd"
```

```ebuild
src_configure() {
	local bzip2_flag=disable-bzip2 zstd_flag=disable-zstd
	use bzip2 && bzip2_flag=-disable-bzip2
	use zstd && zstd_flag=-disable-zstd

	haskell-cabal_src_configure \
		$(cabal_flag dev dev) \
		${bzip2_flag} \
		${zstd_flag}
}
```